src/labels/CustomLabels.labels

```diff
@@ -5244,6 +5244,14 @@ To view these Jobs, search for Scheduled Jobs in Salesforce Setup.
 &lt;br/&gt;
 Selecting Don't Auto Schedule Default NPSP Jobs tells NPSP not to check for or reschedule the default Jobs.</value>
     </labels>
+    <labels>
+        <fullName>stgEnableIncrementalRollupsHelpText</fullName>
+        <categories>Settings</categories>
+        <language>en_US</language>
+        <protected>true</protected>
+        <shortDescription>Enable Incremental Rollups Help Text</shortDescription>
+        <value>ENABLE_INCREMENTAL_ROLLUPS_HELP_TEXT</value>
```

Comment:
@force2b @njjc, I put this placeholder here for the help text (to be provided by doc)